__test__/game.test.js

```diff
@@ -0,0 +1,108 @@
+const Game = require('../src/game').default
+const fs = require('fs')
+
+describe('App', () => {
+  it('Contains the compiled JavaScript', async (done) => {
+    fs.readFile('./public/main.js', 'utf8', (err, data) => {
+      expect(err).toBe(null)
+      expect(data).toMatchSnapshot()
+      done()
+    })
+  })
+})
+
+describe('Game', () => {
+  let game, p1, p2
+  beforeEach(() => {
+    p1 = 'Salem'
+    p2 = 'Nate'
+    game = new Game(p1, p2)
+  })
+
+  describe('Game', () => {
+    it('Initializes with two players', async () => {
+      expect(game.p1).toBe('Salem')
+      expect(game.p2).toBe('Nate')
+    })
+
+    it('Initializes with an empty board', async () => {
+      for (let r = 0; r < game.board.length; r++) {
+        for (let c = 0; c < game.board[r].lenght; c++) {
+          expect(game.board[r][c]).toBeUndefined()
+        }
+      }
+    })
+
+    it('Starts the game with a random player', async () => {
+      Math.random = () => 0.4
+      expect(new Game(p1, p2).player).toBe('Salem')
+
+      Math.random = () => 0.6
+      expect(new Game(p1, p2).player).toBe('Nate')
```

Comment:
Fails

Reply:
Game › Game › Starts the game with a random player

    expect(received).toBe(expected) // Object.is equality

    Expected: "Nate"
    Received: "Bananas"